evalai/get_token.py

```diff
@@ -1,10 +1,10 @@
-import click
 import os
+import click
```

Comment:
@aaaakshat can you reorder this lexicographically? import click should be the first import

Reply:
@Ram81 done on latest commit.